ntbs-service/Pages/Shared/_Footer.cshtml

```diff
@@ -1,31 +1,38 @@
-                            TB Strategy
-                        </a>
-                    </li>
-                    <li class="govuk-footer__inline-list-item">
-                        <a class="govuk-footer__link footer-link" href="https://www.gov.uk/government/publications/tuberculosis-in-england-annual-report" target="_blank" rel="noopener noreferrer">
-                            Annual report
-                        </a>
-                    </li>
-                    <li class="govuk-footer__inline-list-item">
-                        <a class="govuk-footer__link footer-link" href="https://fingertips.phe.org.uk/profile/tb-monitoring" target="_blank" rel="noopener noreferrer">
-                            Fingertips
-                        </a>
-                    </li>
-                    <li class="govuk-footer__inline-list-item">
-                        <a class="govuk-footer__link footer-link" href="#3">
-                            Accessibility Statement
-                        </a>
-                    </li>
-                </ul>
-
+                <div class="footer-list-and-release">
+                    <ul class="govuk-footer__inline-list footer-link-list">
+                        <li class="govuk-footer__inline-list-item">
+                            <a class="govuk-footer__link footer-link" href="https://www.england.nhs.uk/tuberculosis-strategy-for-england-2015-2020/" target="_blank" rel="noopener noreferrer">
+                                TB Strategy
+                            </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                            <a class="govuk-footer__link footer-link" href="https://www.gov.uk/government/publications/tuberculosis-in-england-annual-report" target="_blank" rel="noopener noreferrer">
+                                Annual report
+                            </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                            <a class="govuk-footer__link footer-link" href="https://fingertips.phe.org.uk/profile/tb-monitoring" target="_blank" rel="noopener noreferrer">
+                                Fingertips
+                            </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                            <a class="govuk-footer__link footer-link" href="#3">
+                                Accessibility Statement
+                            </a>
+                        </li>
+                    </ul>
+                    <p class="govuk-footer__link footer-link footer-version">
+                        Release-@VersionService.GetReleaseVersion().Version-@VersionService.GetReleaseVersion().Date.ToShortDateString()
+                    </p>

```

Comment:
Was there a reason not to include this as a further line item in the above list?

Reply:
Mainly spacing, I had a few different versions but basically adding the margin to the item in the list before this meant that the spacing wasn't as nice when viewed zoomed in / in a smaller window

Reply:
Do we need an extra margin though?

I guess it depends what the requirements are - I'd prefer to keep things simple and have it as another line item, but if Nancy is keen to add the extra spacing then this is fine. Please could you check with her before you merge.

Reply:
Nancy says she prefers how it is now. It will be easy to add it to the list if we decide to do that in the future.